main/src/test/java/org/mobilitydata/gtfsvalidator/validator/StopZoneIdValidatorTest.java

```diff
@@ -74,8 +74,19 @@ public void emptyFareRuleZoneId_noNotice() {
     assertThat(
             generateNotices(
                 ImmutableList.of(
+                    createStop(4, GtfsLocationType.STOP, "zone id value", "stop id value"),
```

Comment:
For the whole file: please write one unit test per (location type, presence of zone_id). This will give you 5 * 2 = 10 tests.

Reply:
Addressed in https://github.com/MobilityData/gtfs-validator/pull/972/commits/f836516e35f3f3bc711683039a1bff7930477bf9.